spec/features/user_can_reset_password_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+feature 'User can reset password' do
+    before do
+      visit new_user_password_path
+    end
+  
+    describe 'happy path' do
+      let(:registered_user) { create(:user) }
```

Comment:
can you extract this to the before action?

Reply:
I tried this, but I keep getting errors such as 

```
1) User can reset password happy path expect user to able to reset password
     Failure/Error: let(:registered_user) { create(:user) }
       `let` is not available from within an example (e.g. an `it` block) or from constructs that run in the scope of an example (e.g. `before`, `let`, etc). It is only available on an example group (e.g. a `describe` or `context` block).
     # ./spec/features/user_can_reset_password_spec.rb:6:in `block (2 levels) in <top (required)>'
```

so that's why it's in each describe blocks

Reply:
moved it up to just beneath feature row. This worked. 
now we only call for the user creation one time instead of 2.